test/promql/tests/data_plane/release_service_test.yaml

```diff
@@ -0,0 +1,65 @@
+rule_files:
+  - prometheus.release_service_alerts.yaml
+
+evaluation_interval: 1m
+
+tests:
```

Comment:
All recording rules are using `sum`, but no test case tests that.

Reply:
hum. I understood that test case would only need the raw data and the positive (or negative) case and the sum would occur only in the alert rule itself. Why a sum test case would be required? Could you clarify?

Reply:
I see.